src/commands/admin/jobs/stop.js

```diff
@@ -38,11 +39,13 @@ module.exports = ({
       }, {
         guildId: guild.id,
         jobName: job.meta.name,
-        jobState: false
+        jobState: false,
+        jobInterval: null,
```

Comment:
We probably don't want to reset the guild owners set interval here, we could probably just leave it as it is instead of resetting it.